rails_programming/rails_basics/web_refresher.md

```diff
@@ -132,3 +144,18 @@ This section contains helpful links to other content. It isn't required, so cons
 
 * [HTTP explained by Harvard's David Malan](http://www.youtube.com/watch?v=8KuO4r5CHjM)
 * [HTTP Request/Response Basics](http://justahelp.blogspot.com/2013/09/http-requestresponse-basics.html) from Pralay Roy
+
+### Knowledge Check
+This section contains questions for you to check your understanding of this lesson. If you’re having trouble answering the questions below on your own, review the material above to find the answer.
+
+* <a class="knowledge-check-link" href="https://code.tutsplus.com/tutorials/http-the-protocol-every-web-developer-must-know-part-1--net-31177">How do you call an HTTP-message that goes from client to server? How do you call one that goes the opposite direction? In which header would you encounter a "status code" and in which one an "action verb"?</a>
+* <a class="knowledge-check-link" href="https://code.tutsplus.com/tutorials/http-the-protocol-every-web-developer-must-know-part-1--net-31177">What is the difference between HTTP and HTTPS?</a>
```

Comment:
I'd drop this question because it isn't very relevant and the linked resource only barely touches on it anyways.